content/about/nightlies/_index.md

```diff
@@ -21,7 +21,7 @@ type = "nightlies"
 #### Minimum
 * 64-bit CPU & OS
 * Memory: 2GB Ram
-* Graphics: OpenGL 3.3 or greater
+* Graphics: DirectX 9.0c
```

Comment:
This description is specific to Windows. Do "OpenGL ES2 / DirectX 9.0c" here, and "OpenGL 3.3 / DirectX 11 or greater" for recommended.